pandas/tslib.pyx

```diff
@@ -2722,6 +2722,13 @@ class Timedelta(_Timedelta):
     __pos__ = _op_unary_method(lambda x: x, '__pos__')
     __abs__ = _op_unary_method(lambda x: abs(x), '__abs__')
 
+
+# Resolution is in nanoseconds
+# (2**63)/(1*24*60*60*(10**9)) = 106751.991167
+Timedelta.min = Timedelta(-106751.991167, 'D')
```

Comment:
Can we specify this in `ns` instead of fractional days, which might have rounding errors? e.g., `Timedelta(np.info(np.int64).min, 'ns')`